acoustid/data/track.py

```
@@ -5,7 +5,7 @@
 import uuid
 from typing import Any, Dict, Iterable, List, Optional, Set, Tuple, Union
 
-from sqlalchemy import Column, Table, sql
+from sqlalchemy import Column, Table, sql, Row
```

Ruff (0.8.2)

`sqlalchemy.Column` imported but unused

Remove unused import

(F401)